samples/net/echo_client/boards/nrf52840_pca10056.conf

```diff
@@ -0,0 +1,33 @@
+CONFIG_BT=n
+
+CONFIG_NET_TX_STACK_SIZE=4096
+
+CONFIG_REBOOT=y
+
+CONFIG_NET_LOG=y
+CONFIG_SYS_LOG_SHOW_COLOR=y
+CONFIG_PRINTK=y
+
+CONFIG_UART_INTERRUPT_DRIVEN=y
+
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_MPU_ALLOW_FLASH_WRITE=y
+
+CONFIG_NET_IPV4=n
```

Comment:
Original config file for 802154 nRF does not enable TCP, therefore we should disable it here -> `CONFIG_NET_TCP=n`